src/test/java/org/ays/tests/institution/assignmentmanagementservice/PostAssignmentTest.java

```diff
@@ -95,7 +95,7 @@ public void createAssignmentWithInvalidLineNumber(String lineNumber) {
         }
     }
 
-    @Test(dataProvider = "invalidLongitudeValues")
+    @Test(dataProvider = "invalidLongitudeValues",dataProviderClass = DataProvider.class)
```

Comment:
Kod formatında problem var. Bu tür durumlar yaşamamak için aşağıdaki dokümanı inceleyebilirsiniz;
https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html